containerfiles/unstable

```diff
@@ -320,6 +320,15 @@ COPY rootfs/usr/share/polkit-1/rules.d/50-one.playtron.rpmostree1.rules			/usr/s
 # Custom playtron weston session file
 COPY rootfs/usr/share/wayland-sessions/playtron-weston.desktop				/usr/share/wayland-sessions/
 
+# Configure SELinux.
+RUN mkdir -p /usr/share/selinux/playtron-modules
+COPY rootfs/usr/share/selinux/playtron-modules/dontaudit-powerprofiles_t-passwd_file_t.te	/usr/share/selinux/playtron-modules/
+RUN cd /usr/share/selinux/playtron-modules/ && checkmodule -M -m -o dontaudit-powerprofiles_t-passwd_file_t.mod dontaudit-powerprofiles_t-passwd_file_t.te && semodule_package -o dontaudit-powerprofiles_t-passwd_file_t.pp -m dontaudit-powerprofiles_t-passwd_file_t.mod
+COPY rootfs/usr/share/selinux/playtron-modules/systemd_timedated_t-etc_t-lnk_file.te	/usr/share/selinux/playtron-modules/
+RUN cd /usr/share/selinux/playtron-modules/ && checkmodule -M -m -o systemd_timedated_t-etc_t-lnk_file.mod systemd_timedated_t-etc_t-lnk_file.te && semodule_package -o systemd_timedated_t-etc_t-lnk_file.pp -m systemd_timedated_t-etc_t-lnk_file.mod
+RUN semodule -i /usr/share/selinux/playtron-modules/*.pp
```

Comment:
Should we also remove the intermediate `.te` and `.mod` files in the final build?